src/technologies/h.json

```
@@ -1788,11 +1788,39 @@
     "scriptSrc": "/v1/lst/universal-script\\?ph\\=",
     "website": "https://hyros.com"
   },
+  "Hyva Checkout": {
+    "cats": [
+      6
+    ],
+    "description": "A Checkout built for Hyvä Themes using Magewire, Alpine.js and Tailwind CSS.",
+    "js": {
+      "hyvaCheckout": ""
+    },
+    "headers": {
+      "X-Built-With": "^Hyva Checkout$"
+    },
+    "icon": "Hyva Themes.svg",
+    "implies": [
+      "Magento\\;version:2",
+      "Tailwind CSS",
+      "Alpine.js",
+      "Magewire"
+    ],
+    "pricing": [
+      "recurring",
+      "low"
+    ],
+    "website": "https://www.hyva.io/hyva-checkout.html"
+  },
   "Hyva Themes": {
     "cats": [
+      6,
       108
     ],
     "description": "Hyva Themes is a performance-optimised theme for Magento 2 which eliminated the third-party libraries and having only two dependencies Alpine.js and Tailwind CSS.",
+    "js": {
+      "window.hyva": ""
```

I don't think you need the `window` sinc ethis only checks global objects anyway.

```suggestion
      "hyva": ""
```

---

updated in https://github.com/HTTPArchive/wappalyzer/pull/96/commits/b983d6fe703a9618ff2ec0968a7ae4784740055d